doc/keycmds.dsv

```diff
@@ -10,8 +10,8 @@ save||s||Save the currently selected article to a file.
 save-all||n/a||Save all articles from currently selected feed.
 next-unread||n||Jump to the next unread article.
 prev-unread||p||Jump to the previous unread article.
-next||J||Jump to next article.
-prev||K||Jump to previous article.
+next||J||Jump to next article or list entry.
+prev||K||Jump to previous article or list entry.
```

Comment:
I don't think there is anything inherently special about articles—it just so happened that they were mentioned in this doc originally, even though `next`/`prev` could be applied to feeds, tags, URLs etc. I think it'd be fine to unify them all under "list entry", and just write "Jump to next/previous list entry".

Reply:
You're right, I agree, done